pkg/log/log.go

```diff
@@ -146,15 +146,15 @@ func configure(options *Options, b builder) error {
 	return nil
 }
 
-// Debug outputs a message at debug level.
-// This call is a wrapper around [Logger.Debug](https://godoc.org/go.uber.org/zap#Logger.Debug)
-func Debug(msg string, fields ...zapcore.Field) {
+// Debugx outputs a message at debug level.
```

Comment:
To me it feels the most intuitive name for these functions is without any extra suffix. `Debug` for example. Should we leave `Debug` as the high performant function and name others whatever we want.. It will make the user do the natural thing first and will add the high performant code everywhere. Thoughts ?